scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1461,109 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def match_vectors(cls, a, b, weights=None, normalized=False):
+        """Estimate the rotation to match two sets of vectors.
+
+        Find a rotation between frames A and B which best matches a set of unit
+        vectors `a` and `b` observed in these frames. The following loss
+        function is minimized to solve for the direction cosine matrix
+        :math:`C`:
+
+        .. math::
+
+            L(C) = \\frac{1}{2} \\sum_{i = 1}^{n} w_i \\lVert \\mathbf{a}_i -
+            C \\mathbf{b}_i \\rVert^2 ,
+
+        where :math:`w_i`'s are the `weights` corresponding to each vector.
+
+        The rotation is estimated using Markley's SVD method [1]_.
+
+        Parameters
+        ----------
+        a : array_like, shape (N, 3)
+            Vectors expressed in initial frame A. Result of applying an
+            arbitrary rotation to `b`. Each `a[i]` corresponds to a vector.
+        b : array_like, shape (N, 3)
+            Vectors expressed in another frame B. Shape must match `a`. Each
```

Comment:
"Vectors in frame B. Each row of `b` denotes a vector" is sufficient. You already established the A-B difference in the text above. Shape match is also given by the  shape hence redundant.

Reply:
Placed Just before the new commit. Timed perfectly :)